modules/ROOT/pages/update-acm.adoc

```diff
@@ -53,3 +53,18 @@ Attempting to upgrade the package without this feature enabled causes the proces
 This starts the installation process.
 
 When the upgrade finishes, you receive a notification email.
+
+After installing, ensure both `Member` and `Guest` profiles have field-level `Read` access to the `ParentId` field in the `CommunityAPI` object:
```

Comment:
This is only required when upgrading from version 1.62 or below.

Reply:
Thank you! Updated with this info.